tests/unit/auth.tcl

```diff
@@ -55,7 +70,6 @@ start_server {tags {"auth_binary_password external:skip"}} {
     } {WRONGPASS*}
 
     test {AUTH succeeds when binary password is correct} {
-        r config set requirepass "abc\x00def"
         r auth "abc\x00def"
     } {OK}
```

Comment:
I think this test will fail if if you run it with `runtest --single unit/auth --only 'AUTH succeeds when binary password is correct'` because `config set` is inside the other test case.

To fix it, we can to do `r config set requirepass "abc\x00def"` outside of both test cases.